src/type_translator.ts

```diff
@@ -240,7 +240,7 @@ export class TypeTranslator {
     }
 
     const name = this.typeChecker.symbolToEntityName(
-        sym, ts.SymbolFlags.None, this.node, ts.NodeBuilderFlags.UseFullyQualifiedType);
+        sym, ts.SymbolFlags.Type, this.node, ts.NodeBuilderFlags.UseFullyQualifiedType);
```

Comment:
What is this for?

Reply:
I was trying to fix the emit problem mentioned in the description. While doing that I learned that we should set `Type` here, because that's how we want to use the name (as opposed to as a `Value`). It doesn't have any effect though, just an API use cleanup.